package.json

```diff
@@ -81,7 +83,7 @@
     "jsdom": "^22.1.0",
     "mocha": "^10.0.0",
     "mq-polyfill": "^1.1.8",
-    "msw": "^1.3.2",
+    "msw": "^2.2.1",
```

Comment:
It's necessary to upgrade to support `URL` objects in `fetch`, but the upgrade itself is pretty involved, so I'll likely split that out into a separate pull request.

Reply:
> It's necessary to upgrade to support `URL` objects in `fetch`, but the upgrade itself is pretty involved, so I'll likely split that out into a separate pull request.

Addressed separately in #10123.